.github/workflows/openjdk_14_build.yml

```diff
@@ -0,0 +1,117 @@
+name: openjdk_14_build
+on:
+  pull_request:
+  push:
+    branches: [ main, develop ]
+    tags: [ v* ]
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    container: gcr.io/swirlds-registry/cci-openjdk-ghr:14.0.2
+    env:
+      MAVEN_OPTS: -Xmx5200m
+      LC_ALL: C.UTF-8
+      DEBIAN_FRONTEND: noninteractive
+    timeout-minutes: 60
+    steps:
+      - run: apt-get -y install software-properties-common
+      - run: add-apt-repository ppa:git-core/ppa
+      - run: apt-get -y install git
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - run: mvn --version
+      - name: Cache SonarCloud packages
+        uses: actions/cache@v1
+        with:
+          path: ~/.sonar/cache
+          key: ${{ runner.os }}-sonar
+          restore-keys: ${{ runner.os }}-sonar
+      - name: Cache Maven packages
+        uses: actions/cache@v1
+        with:
+          path: ~/.m2
+          key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-m2
+      - name: Compile
+        run: mvn -DskipTests clean compile
```

Comment:
Would highly suggest installing the [Maven Wrapper](https://maven.apache.org/wrapper/) into your repository so it provides repeatable builds. It guarantees CI and local executions are on the same maven version and you can provide JVM and maven configuration in dedicated source controlled [files](https://github.com/hashgraph/hedera-mirror-node/tree/main/.mvn) instead of as `MAVEN_OPTS: -Xmx5200m`. You would then just execute `./mvnw` instead of `mvn` everywhere.

Reply:
I'll add a backlog item to look into this.

Reply:
https://github.com/hashgraph/hedera-transaction-tool-demo/issues/258

Reply:
nevermind, doing this